httpapi/httpapi.go

```diff
@@ -17,11 +17,26 @@ type HTTPAPI struct {
 // RegisterRoutes register HTTP API routes
 func (h HTTPAPI) RegisterRoutes(router *httprouter.Router) {
 	router.Handler("GET", "/v1/metrics", promhttp.Handler())
+	router.GET("/v1/spaces/:space/connections", h.listConnections)
 	router.POST("/v1/spaces/:space/connections", h.createConnection)
 	router.PUT("/v1/spaces/:space/connections/:id", h.updateConnection)
 	router.DELETE("/v1/spaces/:space/connections/:id", h.deleteConnection)
 }
 
+func (h HTTPAPI) listConnections(w http.ResponseWriter, r *http.Request, params httprouter.Params) {
+	w.Header().Set("Content-Type", "application/json")
+	encoder := json.NewEncoder(w)
+
+	space := params.ByName("space")
+	connections, err := h.Connections.ListConnections(space)
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		encoder.Encode(&Response{Errors: []Error{{Message: err.Error()}}})
+	} else {
```

Comment:
Slight change, would outdent this here and add a `return` to the `if` block

Reply:
@sebito91 done